index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Kelvin &mdash;portfolio</title>
+</head>
+<body>
+
+<nav class="nav_container">
+    <h3>Kely Bush</h3>
+    <img src="assets/nav-icon.svg" alt="Toggle Menu">
+</nav>
+<section class="header">
+    <h1>I'm Kelvin Glad to see you!</h1>
+    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+    <h4>Let's Connect</h4>
+    <div class="header_socials">
+        <a href="https://twitter.com/kelvin_bush_">
+            <img src="assets/twitter.svg" alt="Twitter Icon"></a>
+        <a href="https://www.linkedin.com/in/kelvin-wachiye-04b469173/">
+            <img src="assets/linkedin-icon.svg"
+                 alt="LinkedIn Icon"></a>
+        <a class="medium-icon" href="https://github.com/kelvinbush">
+            <img src="assets/medium-icon.svg" alt="Medium Icon"></a>
+        <a href="https://github.com/kelvinbush">
+            <img src="assets/github.svg" alt="Github Icon"></a>
+        <a href="https://github.com/kelvinbush">
+            <img src="assets/cool-icon.svg" alt="Cool Icon"></a>
+    </div>
```

Comment:
- [ ] You are doing great displaying the social icons, You should place the social icons in a `<ul>` tag and each icon should be in an `<a>` tag inside of the `<li>` tag. Please fix this by using the `<ul>` list item element.

Reply:
Adressed 👍🏻